app/_hub/kong-inc/ai-semantic-prompt-guard/how-to/_index.md

```diff
@@ -0,0 +1,176 @@
+---
+nav_title: Using the AI Semantic Prompt Guard plugin
+title: Using the AI Semantic Prompt Guard plugin
+---
+
+The AI Semantic Prompt Guard configuration takes two arrays of objects: one for `allow` prompts, and
+one for `deny` prompts.
+
+## Prerequisites
+
+First, as in the [AI Proxy](/hub/kong-inc/ai-proxy/) documentation, create a service, route, and `ai-proxy` plugin
+that will serve as your LLM access point.
```

Comment:
We don't need an ai-proxy to use the semantic-prompt-guard plugin. But it's fine to add one in example. Not sure how should we phrase this.